po/sv.po

```diff
@@ -549,39 +548,34 @@ msgstr ""
 "nätverkstid ska vara aktiverat."
 
 #: ../src/core/dbus-unit.c:428
-#, fuzzy
 msgid "Authentication is required to start '$(unit)'."
-msgstr "Autentisering krävs för ange systemtiden."
+msgstr "Autentisering krävs för att starta '$(unit)'."
```

Comment:
Usually `'` are changed into `”` in Swedish translations unless the string used by the program is unable to cope with this. I believe that `”` might work, right..?